src/Interpreters/InterpreterShowTablesQuery.cpp

```diff
@@ -116,6 +116,32 @@ String InterpreterShowTablesQuery::getRewrittenQuery()
         return rewritten_query.str();
     }
 
+    /// SHOW MERGES
+    if (query.merges)
+    {
+        WriteBufferFromOwnString rewritten_query;
+        rewritten_query << "SELECT table, database, round((elapsed * (1 / progress)) - elapsed, 2) AS estimate_complete, elapsed, "
+                           "round(progress, 2) AS progress, is_mutation, formatReadableSize(total_size_bytes_compressed) AS size, "
+                           "formatReadableSize(memory_usage) AS mem FROM system.merges";
+
+        if (!query.like.empty())
+        {
+            rewritten_query
+                << " WHERE table "
+                << (query.not_like ? "NOT " : "")
+                << (query.case_insensitive_like ? "ILIKE " : "LIKE ")
+                << DB::quote << query.like;
+        }
+
+        /// (*)
+        rewritten_query << " ORDER BY elapsed desc, database, table";
```

Comment:
Do we really need `database, table` as part of the query? Isn't `elapsed` enough?

Reply:
Have changed, only elapsed remained.